merkletree/proof_test.go

```diff
@@ -53,6 +53,13 @@ func TestVerifyProof(t *testing.T) {
 
 	// proof of inclusion
 	proof := m.Get(index3)
+	if proof.Leaf.Value() == nil {
+		t.Fatal("Expect returned leaf's value is not nil")
+	}
+	// ensure this is a proof of inclusion
+	if !bytes.Equal(proof.LookupIndex, proof.Leaf.Index()) {
```

Comment:
Do you mean this check `proof.LookupIndex == privateIndex(name)` is  
https://github.com/coniks-sys/coniks-go/blob/c04f1e2d88fe5edc35add235942a7b95162d8fe9/merkletree/proof_test.go#L64 ?